controller/gen/config/config.pb.go

```diff
@@ -182,6 +182,7 @@ type Proxy struct {
 	DisableExternalProfiles bool                  `protobuf:"varint,12,opt,name=disable_external_profiles,json=disableExternalProfiles,proto3" json:"disable_external_profiles,omitempty"`
 	ProxyVersion            string                `protobuf:"bytes,13,opt,name=proxy_version,json=proxyVersion,proto3" json:"proxy_version,omitempty"`
 	ProxyInitImageVersion   string                `protobuf:"bytes,14,opt,name=proxy_init_image_version,json=proxyInitImageVersion,proto3" json:"proxy_init_image_version,omitempty"`
+	WaitBeforeExitSeconds   int32                 `protobuf:"varint,15,opt,name=wait_before_exit_seconds,json=waitBeforeExitSeconds,proto3" json:"wait_before_exit_seconds,omitempty"`
```

Comment:
Could someone help me to find a documentation about the line format?

Reply:
You need to change only the file `/proto/config/config.proto`
```diff
diff --git a/proto/config/config.proto b/proto/config/config.proto
index 64b05547..3285e574 100644
--- a/proto/config/config.proto
+++ b/proto/config/config.proto
@@ -50,6 +50,8 @@ message Proxy {
   string proxy_version = 13;
 
   string proxy_init_image_version = 14;
+
+  int32 wait_before_exit_seconds = 15;
 }
 
 message Image {
```
and then run `bin/protoc-go.sh` which will regenerate the `pb.go` files automatically

Reply:
Oh.. got it, thank you.

Reply:
This is no longer needed, as per my comment below.